Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonForm.cs

```diff
@@ -244,7 +244,91 @@ protected override void Dispose(bool disposing)
         }
         #endregion
 
-        #region Public
+        #region Public (New)
```

Comment:
Please update the Yer the file has been changed. (To obey the BSD clause requirements)